README.md

```diff
@@ -47,6 +47,12 @@ npm run build
 
 This will produce the js bundle along with assets inside the `./dist` directory
 
+```sh
+npx nodegui-packer --init MyApp
```

Comment:
You do not need to run the init command again when bundling an updated version of your app. 
Step 2 is the only step you need to do after you update your code. 

If you run init again then you loose all the changes your made like app icons, name,etc.